contrib/depends/packages/freebsd_base.mk

```
@@ -1,6 +1,6 @@
 package=freebsd_base
 $(package)_version=11.3
```

```suggestion
$(package)_version=14.1
```
11.3 EOL in 2020. Maybe bump? Idk

---

Version bump PR here: #9453